features/flows/rest/impl/src/main/java/org/opennms/netmgt/flows/rest/internal/classification/ClassificationRestServiceImpl.java

```diff
@@ -133,7 +133,7 @@ public Response getRule(int id) {
     public Response saveRule(RuleDTO ruleDTO) {
         final Rule rule = convert(ruleDTO);
         rule.setId(null);
-
+        rule.setPosition(0); // new rules get the lowest position (lowest priority)
```

Comment:
Shouldn't this mean `new rules get the lowest position (highest priority)`?

Reply:
as discussed: the rule at position 0 will be evaluated first, rule on position 2 next, etc. new rules get a high position => will be evaluated last